src/BlockchainTestsFiller/TransitionTests/bcBerlinToLondon/initialValFiller.yml

```diff
@@ -0,0 +1,480 @@
+      maxPriorityFeePerGas: 1000
+      nonce: auto
+      secretKey: 41f6e321b31e72173f8ff2e292359e1862f24fba42fe6f97efaf641980eff298
+      to: 0xCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC
+      value: 0x0
+
+    # Type 2 transaction, tip = maxFeePerGas - baseFee
+    - data: :abi f(uint) 0x6030
+      accessList: []
+      gasLimit: 1000000
+      maxFeePerGas:         1000000000
+      maxPriorityFeePerGas:  500000000
+      nonce: auto
+      secretKey: 41f6e321b31e72173f8ff2e292359e1862f24fba42fe6f97efaf641980eff298
+      to: 0xCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC
+      value: 0x0
+
+    # This type 0 transaction has gasPrice = baseFee+1. It should happen
+    - data: :abi f(uint) 0x6040
+      gasLimit: 1000000
+      gasPrice:  875000045
+      nonce: auto
+      secretKey: 41f6e321b31e72173f8ff2e292359e1862f24fba42fe6f97efaf641980eff298
+      to: 0xCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC
+      value: 0x0
+
+    # This type 0 transaction has gasPrice = baseFee. It should happen
+    - data: :abi f(uint) 0x6050
+      gasLimit: 1000000
+      gasPrice:  875000044
+      nonce: auto
+      secretKey: 41f6e321b31e72173f8ff2e292359e1862f24fba42fe6f97efaf641980eff298
+      to: 0xCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC
+      value: 0x0
+
+
+    # This type 0 transaction has gasPrice < baseFee. It won't happen
+    # Invalid transactions mess up the nonce calculation, so I prefer to do them
+    # with otherwise unused accounts  
+    - data: :abi f(uint) 0x6060
+      expectException:
+        BerlinToLondonAt5: TR_FeeCapLessThanBlocks
+      gasLimit: 1000000
+      gasPrice: 875000043
+      nonce: auto
+      secretKey: 0x30aef158fab404b766b50db4618b2231ba4e39148a563dc69d80927cd620cfe5
+      to: 0xCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC
+      value: 0x0
+
+
+    # This type 1 transaction has gasPrice = baseFee+1. It should happen
+    - data: :abi f(uint) 0x6070
+      accessList: []
+      gasLimit: 1000000
+      gasPrice:  875000045
+      nonce: auto
+      secretKey: 41f6e321b31e72173f8ff2e292359e1862f24fba42fe6f97efaf641980eff298
+      to: 0xCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC
+      value: 0x0
+
+    # This type 1 transaction has gasPrice = baseFee. It should happen
+    - data: :abi f(uint) 0x6080
+      accessList: []
+      gasLimit: 1000000
+      gasPrice:  875000044
+      nonce: auto
+      secretKey: 41f6e321b31e72173f8ff2e292359e1862f24fba42fe6f97efaf641980eff298
+      to: 0xCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC
+      value: 0x0
+
+
+    # This type 1 transaction has gasPrice < baseFee. It won't happen
+    # Invalid transactions mess up the nonce calculation, so I prefer to do them
+    # with otherwise unused accounts  
+    - data: :abi f(uint) 0x6090
+      expectException:
+        BerlinToLondonAt5: TR_FeeCapLessThanBlocks
+      gasLimit: 1000000
+      gasPrice: 875000043
+      nonce: auto
+      secretKey: 0x69d66403e6a54b18c2a768af4a8de554df133550d4dfa37a377ec8c0e85b9ef4
```

Comment:
so the nonce auto wont work here if it was the same secretKey as in valid transactions?

Reply:
It won't work if the same secretKey is used in valid transactions (or possibly invalid ones) later. If this is the last transaction with that secretKey then there's no problem.